MapboxNavigation/Resources/sv.lproj/Localizable.strings

```diff
@@ -1,4 +1,4 @@
-﻿﻿/* Format for speech string; 1 = way name; 2 = distance */
+﻿/* Format for speech string; 1 = way name; 2 = distance */
```

Comment:
@1ec5 Any idea why I have to replace this comment manually with the one in Base-language after pulling in from Transifex and converted to UTF-8?
![screenshot 2017-05-03 13 40 22](https://cloud.githubusercontent.com/assets/764476/25659021/137d2f62-3006-11e7-8dbf-27b48af3dc73.png)